src/webgpu/web_platform/canvas/readbackFromWebGPUCanvas.spec.ts

```diff
@@ -421,6 +440,19 @@ g.test('drawTo2DCanvas')
     - colorSpace = {"srgb", "display-p3"}
     - WebGPU canvas type = {"onscreen", "offscreen"}
     - 2d canvas type = {"onscreen", "offscreen"}
+
+
+    * makes a webgpu canvas with the given colorSpace and puts data in via copy convoluted
+      copy process
+    * makes a 2d canvas with 'srgb' colorSpace (the default)
+    * draws the webgpu canvas into the 2d canvas so if the color spaces do not match
+      there will be a conversion.
+    * gets the pixels from the 2d canvas via getImageData
+    * compares them to hard coded values that are converted to expected values by copying
+      to an ImageData of the given color space, and then using putImageData into an srgb canvas.
+
+      canvas(colorSpace) -> canvas(srgb).drawImage -> canvas(srgb).getImageData -> actual
```

Comment:
Should these be `canvas(colorSpace)` instead of `canvas(srgb)` for all three?

Reply:
No, it's correct. Though the name `expectCanvas` at line 479 should be called `actualCanvas`. Will rename that in this PR.

1. it calls initWebGPUCanvasContent(colorSpace)  
2. it makes `expectCanvas` with no color space so it's the default (srgb)
3. it draws the WebGPU canvas(colorSpace) into expectCanvas(srgb)
4. it calls getImageData on the expectCanvas(srgb)